pages/static/src/js/groups.js

```diff
@@ -141,12 +180,23 @@ module.exports = function() {
 
         // TODO: show recurring meeting here instead
         description = groups[i].properties.description;
+        group_type = groups[i].properties.group_type;
+        
+        if (group_type == 'state-organizing-committee') {
+          group_type = 'State Organizing Committee';
+        } else if (group_type == 'state-chapter') {
+          group_type = 'State Chapter';
+        } else if (group_type == 'campus') {
+          group_type = 'Campus Group'; 
+        } else {
+          group_type = null;
+        }
 
         $('.groups-map-info__status').append('\
           <div class="component">\
             <div class="component__heading">\
-              <span class="component__location">' + location + '</span>\
-              <h4 class="component__name">' + groups[i].properties.name + ((groups[i].properties.state_organizing_committee) ? " (SOC) " : "") + '</h4>\
+              <span class="component__location">' + location + ((group_type) ? ' - ' + group_type : "") + '</span>\
```

Comment:
Is it correct that you are adding this next to location rather than name?

Reply:
@cjmabry just want to double check this is right?

Reply:
Is fine, classname should probably not be called location but is generally just a subheading.